### PR TITLE
Added new translation or fixed the existing.

### PR DESCRIPTION
**Pull request by melih**
Still has errors and missing translations.

### CODE REVIEW

File: publishable/lang/tr/voyager.php [hunk 1]
```diff
@@ -94,74 +94,82 @@
...
-        'successfully_deleted' => 'Başarıyla Ayar silindi ',
+        'add_new'              => 'Yeni Ayar Ekle',
+        'delete_question'      => ' :setting ayarını silmek istediğine emin misin?',
+        'delete_confirm'       => 'Evet, Bu Ayarı Sil',
+        'successfully_created' => 'Başarıyla Ayar Oluşturuldu',
+        'successfully_saved'   => 'Başarıyla Ayar Kaydedildi ',
+        'successfully_deleted' => 'Başarıyla Ayar Silindi ',
         'already_at_top'       => 'Zaten Listenin en üstünde',
         'already_at_bottom'    => 'Zaten listenin en altında',
+        'key_already_exists'   => ':key anahtarı zaten mevcut',
```
Comment (defaultpage) [inline publishable/lang/tr/voyager.php]: Hi! Can you remove [unwanted spaces](https://styleci.io/analyses/XVZVrK)?

Comment (fletch3555) [reply publishable/lang/tr/voyager.php]: StyleCI will auto-fix these issues once it's merged.  I'm not very worried about failed StyleCI checks.

Comment (defaultpage) [reply publishable/lang/tr/voyager.php]: Thanks. I didn't know.

Comment (fletch3555) [reply publishable/lang/tr/voyager.php]: No worries.

**Comment by fletch3555** [comment]
@melihsaricam, we've split out the translation files into multiple files.  Could you please update this PR matching that pattern? You'll need to merge the `1.x-dev` branch into yours, fix the conflicts, then commit and push.

**Comment by marktopper** [comment]
Closed due to missing activity

**Pull request closed without merging**